src/App.tsx

```diff
@@ -15,47 +16,86 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [reverse, setReverse] = useState(false);
+  const [type, setType] = useState(SortType.NONE);
```

Comment:
to make it motre descriptive
```suggestion
  const [sortType, setSortType] = useState(SortType.NONE);
```